monitoring/metrics/in-servicepulse.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Performance Metrics in ServicePulse
+summary: Describes how to use ServicePulse to review endpoint performance metrics
+reviewed: 2018-01-12
+---
+
+ServicePulse collects displays performance monitoring data about running endpoints on the Monitoring tab.
+
+![ServicePulse monitoring tab](servicepulse-monitoring-tab.png)
+
+NOTE: ServicePulse will only display the monitoring tab if it is configured to point to a ServiceControl monitoring instance.
+
+The main monitoring tab shows a list of logical endpoints as well as the performance data collected about those endpoints. If a logical endpoint is running more than one physical instance, a badge will show a count of physical instances.
+
+![ServicePulse monitoring tab instance count](servicepulse-instance-count.png)
```

Comment:
No image?

Reply:
It's there. It had an uppercase extension.